README.md

```diff
@@ -105,6 +105,27 @@ To construct a Market Assessment Time Series the following must be provided.
 
 [Go to Time Extraction window section](#artesian-sdk-extraction-windows)
 
+### Auction Time Series
+```Python
+from Artesian import *
+
+qs = QueryService(cfg);
+data = qs.createAuction() \
+    .forMarketData([100011484,100011472,100011477,100011490,100011468,100011462,100011453]) \
+    .inAbsoluteDateRange("2018-01-01","2018-01-02") \
+    .inTimeZone("UTC") \
+    .execute()
+
+```
+To construct an Auction Time Series the following must be provided.
+<table>
```

Comment:
are you sure Table in Markdown are to be done in this way?

afaik it was done

```
| header1 | header2 |
| --           | --           |
| value     | value      |
```

| header1 | header2 |
| -- | -- |
| value     | value      |

Reply:
actually it works